logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/DeletedConversationEventHandler.kt

```diff
@@ -24,8 +24,8 @@ import com.wire.kalium.logic.data.event.Event
 import com.wire.kalium.logic.data.event.EventLoggingStatus
 import com.wire.kalium.logic.data.event.logEventProcessing
 import com.wire.kalium.logic.data.user.UserRepository
-import com.wire.kalium.logic.feature.message.EphemeralConversationNotification
-import com.wire.kalium.logic.feature.message.EphemeralNotificationsMgr
+import com.wire.kalium.logic.feature.message.ephemeral.EphemeralConversationNotification
+import com.wire.kalium.logic.feature.message.ephemeral.EphemeralNotificationsMgr
```

Comment:
This is not coming from my changes I only moved it into subpackage

Reply:
For future reference, I'd advise to keep the PRs more atomic.

This PR would probably be merged earlier if it was just limited to tests and not moving packages around.

In my personal experience, if I want something important merged soon: I'll raise a PR with only that.

- "Oh, but renaming this will only take me 1 minute!"
- And now people are discussing about naming in my PR that was meant to be merged today 😅 

Renaming, refactoring, moving packages, cleaning in general, can always have their own PR.

It will also be easier for the reviewers to quickly separate between different topics.

Reply:
Also, `EphemeralNotificationsManager` doesn't seem to have nothing to do with ephemeral/self-deleting messages.

It's about "ephemeral notifications". Which I have no clue what it means 😓. But seems to be related to conversations being deleted.
Maybe @yamilmedina or @borichellow can help out.

But I don't think this should be included together in the same package as self-deleting messages, as they seem to be completely unrelated.

So at least `EphemeralNotificationsManager`, `EphemeralNotificationsMgr`, `EphemeralConversationNotification` should be on their own package.

Something like `notifications.ephemeral`, and not `messages.ephemeral`.